general/development/policies/accessibility.md

```diff
@@ -121,7 +123,48 @@ See [the ARIA best practice advice on landmarks](https://www.w3.org/TR/wai-aria-
 
 All pages should have a unique title that describes the current page.
 
-See [the WCAG 2.1 success criteria for web page titles](https://www.w3.org/TR/WCAG21/#page-titled) for further information.
+Some tips for providing a meaningful page title:
+
+- The page title must be accurate and informative.
+- If the page causes a change of context (e.g. a search functionality), it should describe the result or change of context to the user.
+- It should be concise.
+- If possible, it should uniquely identify the page.
+- The most identifying information should come first.
+
+#### Example
+
+Consider that a student in the submission page of an assignment called `Kinetics problem set 1` in the `Physics 101` in the `Moodle University`.
```

Comment:
```suggestion
Consider that a student is on the submission page of an assignment actvity called `Kinetics problem set 1` in the `Physics 101` course in the `Wattle Grove University` Moodle site.
```

Better not to use "Moodle" in the name of the site to improve clarity of the sentence.